### ISSUE TITLE
Failed to minify

### ISSUE THREAD

**Issue by thierryskoda** [Volst/graphql-form-helpers#10]
Hi,

When doing `npm run build` with `create-react-app` I get an error saying:

````
Failed to minify the code from this file:

 	./node_modules/@volst/graphql-form-helpers/dist/parseFormToMutation.js:16
````

And this is where they are pointing me to resolve the issue: [link](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify)

Thank you
Thierry

**Comment by SpaceK33z** [comment]
Ah, I compiled the code down to only ES2017, works for me since I don't have to support old browsers but this isn't really best-practice do to in a package. Anyway, fixed in `@volst/graphql-form-helpers@0.2.4`.

**Comment by thierryskoda** [comment]
Thanks!
